pwiz_tools/Skyline/Model/ImportPeakBoundaries.cs

```diff
@@ -99,6 +99,16 @@ public enum Field { modified_peptide, filename, apex_time, start_time, end_time,
         public static int[] REQUIRED_NO_CHROM { get { return REQUIRED_FIELDS.Take(2).ToArray(); }}
 
         // ReSharper disable LocalizableElement
+
+        public static readonly string[] PEPTIDE_SYNONYMS = new[]
+        {
+            "PeptideModifiedSequence", "ModifiedSequence", "FullPeptideName", "EG.ModifiedSequence", ColumnCaptions.PeptideModifiedSequence, ColumnCaptions.ModifiedSequence
+        };
+        public static readonly string[] MOLECULE_SYNONYMS = new[]

```

Comment:
This should not be a statically initialized array like this, since it contains localized strings (ColumnCaptions.Molecule and ColumnCaptions.MoleculeName).
Maybe make it into a static property that always returns a new array.

Reply:
I'm not sure that's a problem - don't we require a restart when user changes the language setting? Here is is running japanese, in testrunner.exe

![image](https://user-images.githubusercontent.com/1200761/156799984-8def2820-b492-4fa5-8923-81ae3926584f.png)